### PR TITLE
feat: Change RPC types per CKB change

### PR DESCRIPTION
**Pull request by duanyytop**
* Remove several version field
* Change field type long or int to String

### CODE REVIEW

File: core/src/main/java/org/nervos/ckb/methods/type/Header.java [hunk 1]
```diff
@@ -26,17 +25,15 @@
      * "version": 0
      */
 
-    @JsonProperty("cellbase_id")
-    public String cellbaseId;
 
     public String difficulty;
     public String hash;
-    public int number;
+    public String number;
```
Comment (ashchan) [inline core/src/main/java/org/nervos/ckb/methods/type/Header.java]: Please update the comment above as `number` is not a string.

File: core/src/main/java/org/nervos/ckb/methods/type/Header.java [hunk 2]
```diff
@@ -9,7 +9,6 @@
 public class Header {
 
     /**
-     * "cellbase_id": "0x3abd21e6e51674bb961bb4c5f3cee9faa5da30e64be10628dc1cef292cbae324",
      * "difficulty": "0x100",
      * "hash": "0x087c25e23e42f5d1e00e6984241b3711742d5e0eaf75d79a427276473e1de3f9",
      * "number": 1,
```
Comment (ashchan) [inline core/src/main/java/org/nervos/ckb/methods/type/Header.java]:
```suggestion
     * "number": "1",
```

File: core/src/main/java/org/nervos/ckb/methods/type/Header.java (hunk 1 above)
Comment (duanyytop) [reply core/src/main/java/org/nervos/ckb/methods/type/Header.java]: OK

File: core/src/main/java/org/nervos/ckb/methods/type/Header.java (hunk 2 above)
Comment (duanyytop) [reply core/src/main/java/org/nervos/ckb/methods/type/Header.java]: OK

**Comment by duanyytop** [comment]
Updated. Please review again. Thanks

**Comment by ashchan** [comment]
> Updated. Please review again. Thanks

Next time please use `Re-request review` ;-p

**Pull request merged**